test/help.test.js

```diff
@@ -45,6 +46,22 @@ describe('loopback generator help', function() {
     process.env.SLC_COMMAND = undefined;
   });
 
+  it('print right help message for each generator', function() {
+    process.env.SLC_COMMAND = 'loopback --help';
+    var names = ['acl', 'app', 'boot-script', 'datasource', 'export-api-def',
+    'middleware', 'model', 'property', 'relation', 'remote-method', 'swagger'];
+
+    names.forEach(function(name) {
```

Comment:
Please create one test case for each name (command).

In your current implementation, the first failure will abort the whole test and remaining commands are not tested.

Here is a mock-up to show what I mean:

```js
['acl', ..., 'swagger'].forEach(function(cmdName) {
  it('prints correct help message for "yo ' + cmdName + '"', function() {
   var gen = givenGenerator(name, ['--help']);
   // ...
   assert.equal(output, helpText);
  }); 
});